modules/ocf_irc/files/znc.service

```diff
@@ -0,0 +1,11 @@
+[Service]
+ExecStart=/usr/bin/znc --foreground --datadir /opt/znc
+User=znc
+Group=znc
+Restart=always
+StandardOutput=syslog
+StandardError=syslog
+SyslogIdentifier=znc
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
Yeah, I was going to include a reload here, but I read this in the ZNC man page:
```
SIGNALS
       This section explains how znc reacts to different signals:

       SIGINT Exit ZNC. This is equivalent to /znc shutdown

       SIGHUP Reload znc.conf. This is equivalent to /znc rehash.  DO NOT do this
              very often, things can break badly!

       SIGUSR1
              Rewrite znc.conf. This is equivalent to /znc saveconfig
```
It sounds like sending a HUP reloads the config file _without_ saving changes from the web interface since the last write. Is that the right behavior, meaning won't you lose data on a reload that way?

Also, you're right, the stderr/stdout redirects were for testing.